contracts/lifecycle/DestructibleDelayed.sol

```diff
@@ -0,0 +1,78 @@
+pragma solidity ^ 0.4.24;
+
+import "../ownership/Ownable.sol";
+
+/**
+ * @title DestructibleDelayed
+ * @dev Contract that can be selfdestructed after a delay period.
+ */
+
+
+contract DestructibleDelayed is Ownable {
+
+  // Whether a request for the self-destruction of the contract was requested.
+  bool public destructionRequested = false;
+
+  // When self-destruction is possible (UNIX timestamp)
+  uint256 public destructionTime = 2**256-1; 
```

Comment:
Considering adding a comment mentioning this is the same time unit for Solidity's `block.timestamp`.